src/aspire/volume/volume_synthesis.py

```diff
@@ -247,6 +248,9 @@ def _check_order(self):
                 f"An {self.__class__.__name__} must have order=1. Provided order was {self.order}"
             )
 
+    def _set_symmetry_group(self):
```

Comment:
Why is this added here?

Reply:
Prior to adding this, `AsymmetricVolume`'s were instantiated with `CnSymmetryGroup` of `order = 1`. All other asymmetric volumes in the code were using `IdentitySymmetryGroup`, for example wrapping a numpy array in `Volume` or downloading `emdb_2660` using the data downloader. This was just to enforce that all asymmetric volumes were using the `IdentitySymmetryGroup`. I realized this was the case when I tried to use the equality operator `__eq__` (added in this PR) on two asymmetric volumes instantiated in different ways.